Gems/LmbrCentral/Code/include/LmbrCentral/Shape/ShapeComponentBus.h

```diff
@@ -105,11 +114,11 @@ namespace LmbrCentral
     };
 
     /// Services provided by the Shape Component
-    class ShapeComponentRequests : public AZ::ComponentBus
+    class ShapeComponentRequests : public AZ::EBusSharedDispatchTraits<ShapeComponentRequests>
     {
     public:
-        /// allows multiple threads to call shape requests
-        using MutexType = AZStd::recursive_mutex;
+        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
```

Comment:
nitpick: You should make this `constexpr`

Reply:
Fixed!